src/components/elements/side-tab.js

```diff
@@ -36,15 +48,35 @@ const StyledTab = styled.li`
         }
     }
 `
+const StyledDropDown = styled.li`
+    padding: 1rem 0;
+    color: var(--color-red);
+    border-bottom: 1px solid var(--color-red);
+    display: flex;
+    justify-content: space-between;
 
-const Tab = ({ active_tab, label, onClick, text }) => {
+    p {
+        font-size: var(--text-size-s);
+        color: var(--color-red);
+        font-size: 2rem;
+    }
+`
+const PathWrapper = styled(Path)`
+    transform: ${props =>
+        props.what_is_active_tab === '-' ? 'rotate(0deg)' : 'rotate(180deg)'};
```

Comment:
`current_active_tab`?

Reply:
`current_acctive_tab` contains  `active_tab` obejct. so we can access the properties for example `text` and we can use it for conditions or rendering the props.